tests/queries/0_stateless/02346_full_text_search.sql

```diff
@@ -277,74 +277,3 @@ SELECT * FROM tab WHERE str == 'b' AND 1.0;
 --             AND result_rows==1
 --         LIMIT 1;
```

Comment:
It would also be great if you could add one or two sentences of docs about the new parameter to https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/invertedindexes.